wpimath/src/main/java/edu/wpi/first/wpilibj/waypoint/WaypointUtil.java

```diff
@@ -0,0 +1,152 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package edu.wpi.first.wpilibj.waypoint;
+
+import edu.wpi.first.pathweaver.path.Path;
+import edu.wpi.first.pathweaver.path.wpilib.WpilibPath;
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVPrinter;
+import org.apache.commons.csv.CSVRecord;
+
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.io.Reader;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import javafx.geometry.Point2D;
```

Comment:
I don't think this will work on the RIO because there aren't JavaFX packages there. @AustinShalit or @PeterJohnson can confirm however.

Reply:
Yes, there are no javafx packages on the rio. In fact, there are none in any Java 11 distro. So theres no way this would even build.

Reply:
Waypoint also isn't a wpilib thing. It's only in pathweaver, so more would need to be ported.